### DISCUSSION TITLE
openai_api调用消耗显存问题

### DISCUSSION THREAD

**Discussion by zzzengzhe** [zai-org/ChatGLM3#742] [Q&A] [answered]
请问一个python程序连续多次调用openai_api接口的时候消耗的显存是积累的吗？

**Comment by zRzRzRzRzRzRzR** [comment]
不是，取决于你历史占用

**Reply by zzzengzhe** [reply]
好的，谢谢，还有就是我的python程序调用一次接口之后，只要python程序没有关闭，那么显存的占用就不会释放，有没有在不关闭python程序的情况下释放显存？

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
不是啊，你调用不是应该是用API调用的啊，后台的python只要开着显存就会占用

**Reply by zzzengzhe** [reply]
好的，明白了，谢谢
